src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -189,4 +190,13 @@ public function deleteAction(Request $request, Post $post)
 
         return $this->redirectToRoute('admin_post_index');
     }
+
+    /**
+     * Must be protected to allow Symfony DIC to create a proxy.
+     *
+     */
+    protected function getSlugger(): Slugger
```

Comment:
My guess is that this feature will confuse a lot of users. You add to your class an empty getter ... which during runtime is not empty because somehow Symfony injects the needed code to return what you want to get 😕